### ISSUE TITLE
Unable to find services.yaml for the xiaomi_miio_fan integration

### ISSUE THREAD

**Issue by D482** [syssi/xiaomi_fan#113]
installed manually and via HACS, there is no services.yaml file

**Comment by andriej** [comment]
Can confirm, there's warning in 2021.5.0 regarding that.
```
Logger: homeassistant.helpers.service
Source: helpers/service.py:414
First occurred: 21:59:57 (1 occurrences)
Last logged: 21:59:57

Unable to find services.yaml for the xiaomi_miio_fan integration
```

**Comment by syssi** [comment]
Alright. It's not a big thing.

**Comment by vmax77** [comment]
I  think this is solved in 2021.05.1
